source/docs/software/driverstation/manually-setting-the-driver-station-to-start-smartdashboard.rst

```diff
@@ -0,0 +1,37 @@
+Manually Setting the Driver Station to Start SmartDashboard
+===========================================================
+
+.. note:: If WPILib is not installed to the default location (such as when files are copied to a PC manually), the "C++" or "Java" buttons may not properly launch the SmartDashboard. To have the DS start the SmartDashboard when it starts up you have to manually modify the settings for the Default Dashboard.
+
+.. warning:: This is not needed for most installations, try using the appropriate :ref:`Dashboard Type setting <docs/software/driverstation/driver-station:Setup Tab>` for your language first.
+
+Set Driver Station to Default
+-----------------------------
+
+.. image:: images/manually-setting-the-driver-station-to-start-smartdashboard/set-ds-to-default.png
+
+Open the Driver Station software, click on the Setup tab and set the Dashboard setting to Default. **Then close the Driver Station!**
+
+Locate SmartDashboard
```

Comment:
Update to include Shuffleboard as well as SmartDashboard

Reply:
Fixed in 1db7c47.